### ISSUE TITLE
List of allowed file types?

### ISSUE THREAD

**Issue by rowild** [w3c/web-share#200]
Over a couple of days i try to share a geojson file, which seems to throw a "permission denied". 

So I wonder if there is a change to implement some kind of information into the documentation that

- clearly states that there, in fact, is something like "allowed file types" (even the spec mentions "file type" only once; and a user like me might just not think about the fact, that ALLOWED file types have to be considered)
- points at one or more resources where to find, which file types are allowed

**Comment by tomayac** [comment]
For Chromium, this [list](https://docs.google.com/document/d/1tKPkHA5nnJtmh2TgqWmGSREUzXgMUFDL6yMdVZHqUsg/edit), maintained by @ericwilligers, exists.

**Comment by rowild** [comment]
Thank you, @tomayac ! I have this list in my bookmarks.
But I was actually promoting to "increase the usability" of the documentation.

**Comment by marcoscaceres** [comment]
Closing as duplicate of #127. I'm going to paste the link to the list in there...
